tests/core/go_vet_test/BUILD.bazel

```diff
@@ -0,0 +1,42 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test", "go_path", "go_vet_test")
+
+test_suite(
+    name = "go_vet_test",
+)
+
+go_library(
+    name = "lib",
+    srcs = ["lib.go"],
+    importpath = "lib",
+)
+
+go_test(
+    name = "internal_test",
+    size = "small",
+    srcs = ["internal_test.go"],
+    embed = [":lib"],
+)
+
+go_path(
+    name = "lib_srcs",
+    deps = [":lib"],
+)
+
+go_path(
+    name = "all_srcs",
+    deps = [":internal_test"],
+    testonly = True,
+)
```

Comment:
Should `go_path` automatically add the `testonly` attribute if any of the deps have it? Or is that too magical?

Reply:
Too magical.

Reply:
Fair enough.